example/app/config.yml

```diff
@@ -470,6 +470,16 @@ kk:
   ecs:
     ssh: true
     keyPairName: kakelly2-cumulus-sandbox
+  cmr:
+    oauthProvider: launchpad
+    username: CmrUserNotUsed
+    password: CmrPasswordNotUsed
+  launchpad:
+    api: 'https://api.launchpad.nasa.gov/icam/api/sm/v1'
+    passphrase: '{{LAUNCHPAD_PASSPHRASE}}'
+  oauth:
```

Comment:
we want to configure `cumulus-from-source:` deployment to use `launchpad` authentication